src/ui/public/agg_types/buckets/histogram.js

```diff
@@ -47,7 +47,7 @@ define(function (require) {
           write: function (aggConfig, output) {
             var val = aggConfig.params.extended_bounds;
 
-            if (val.min != null || val.max != null) {
+            if (aggConfig.params.min_doc_count) {
```

Comment:
Oops, this shouldn't be here. I have another PR that addresses this issue in #5960. I'll take it out.